runbotics/runbotics-scheduler/src/scheduler-database/tags/tag.service.ts

```diff
@@ -16,7 +16,7 @@ export class TagService {
         private readonly tagRepository: Repository<Tag>
     ) {}
 
-    async getAll(tenantId: string, searchPhrase?: string) {
+    async getAll(tenantId: string, searchPhrase: string | undefined) {
```

Comment:
what is the reason? shouldn't it stay as it was?

Reply:
so far I know, `?` means that argument is optional, so it is unnecessary to pass to the function, but `string | undefined` means that argument must be passed but can be undefined

Reply:
correct, I just wonder why is it required

Reply:
I think we should pass it to the function only if it is undefined